crypto/fipsmodule/ec/p256-nistz.c

```diff
@@ -231,12 +232,13 @@ static void ecp_nistz256_windowed_mul(const EC_GROUP *group, P256_POINT *r,
   ecp_nistz256_point_double(&row[16 - 1], &row[8 - 1]);
 
   BN_ULONG tmp[P256_LIMBS];
-  alignas(32) P256_POINT h;
+  stack_align_type buffer_h[32 + sizeof(P256_POINT)];
```

Comment:
I know this code was here before, but just curious, do you know why is the table aligned to 64  bytes but `h` to 32 bytes?

Reply:
Good question - I never looked into why.